plaso/analysis/browser_search.py

```diff
@@ -57,6 +57,8 @@ class BrowserSearchPlugin(interface.AnalysisPlugin):
        '_ExtractGoogleSearchQuery'),
       ('Google Sites', re.compile(r'sites.google.com/site'),
```

Comment:
I know this is not related to your changes, but could you update `sites.google.com` as well to `sites\.google\.com` and check if there are more URL without proper escaping. Thx in advance.